sysl2/sysl/seqs.go

```diff
@@ -184,7 +174,7 @@ func DoGenerateSequenceDiagrams(stdout, stderr io.Writer, flags *flag.FlagSet, a
 	log.Debugf("no_activations: %t\n", *no_activations)
 	log.Debugf("endpoint_format: %s\n", *endpoint_format)
 	log.Debugf("app_format: %s\n", *app_format)
-	log.Debugf("blackbox: %s\n", blackboxes_flag)
+	log.Debugf("blackbox: %s\n", *blackboxes_flag)
```

Comment:
how do i run at debug level? there should be an option to change the log level.

Reply:
add "log" command line arg to handle this.